docs/eventing/samples/apache-rocketmq-source/config/200-serviceaccount.yaml

```diff
@@ -0,0 +1,21 @@
+#
```

Comment:
I think all this files shouldn't be here. They should be hosted in your repo. I think the install instructions already have links tot them there? For example: https://raw.githubusercontent.com/apache/rocketmq-externals/master/rocketmq-knative/source/config/200-serviceaccount.yaml

Reply:
Thanks for your suggestions, I will delete these files.

Reply:
done